src/main/java/config/SettingConfigFile.java

```diff
@@ -0,0 +1,30 @@
+package config;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+public class SettingConfigFile {
+
+    public Properties setConfig() {
+        Properties properties = new Properties();
+        InputStream input = null;
+        try {
+            input = new FileInputStream("config.properties");
+            properties.load(input);
+            return properties;
+        } catch (IOException exception) {
+            exception.printStackTrace();
```

Comment:
Review this implementation, if we catch an exception we need to do something.
only print the exception is a bad practice. if you are not going to do anything with the exception you should add to the method signature